process/profiles.py

```diff
@@ -155,15 +153,10 @@ def ncore(rhopedn, nped, nsep, nav, alphan):
             )
         )
 
-        if ncore < 0:
-            # Prevent ncore from going negative (and terminating the optimisation) by
-            # kludging to small positive value. Allows solver to continue and
-            # hopefully be constrained away from this point (e.g. constraint 81, ne0 > neped)
-            logger.exception("ncore is negative. Kludging to 1e-6.")
-            ncore = 1.0e-6
-            # raise ValueError(
-            #     f"Core density is negative: {ncore}. {nped = }, {nsep = }, {nav = }"
-            # )
+        if ncore < 0.0:
+            # Allows solver to continue and
+            # warns the user to raise the lower bound on dene if the run did not converge
+            error_handling.report_error(282)
```

Comment:
Running the files it dosent seem to make a difference as they all fail eventually anyway. It seems better to have it fail quickly and the user updates the lower bound to prevent it going negative again in future runs

Reply:
I probably need @jonmaddock to comment on this, as he is the one that wrote the kludge. For now, since this is a level 2 error, I will approve it but will make a follow-up issue to check that this fix is ok with the UQ tools (which is why the kludge was added)

Reply:
If needs be we could put the kludge back in without adding back the logger output as it just slows down the terminal output

Reply:
Probably a good idea, and then #3059 can address making this model more robust